.github/workflows/java-gradle-plugin.yaml

```diff
@@ -65,44 +65,31 @@ on:
         required: true
 
 jobs:
-  build:
-    name: Build
+  java-gradle-base:
+    name: Build, Test and Assess code quality
     runs-on: ubuntu-22.04
 
     steps:
-      - name: Build
-        uses: bakdata/ci-templates/actions/java-gradle-build@v1.16.0
+      - name: Build, Test & Assess
+        uses: bakdata/ci-templates/.github/workflows/java-gradle-base.yaml@feat/split-test
```

Comment:
Why are you including the workflow as a step instead of a job here? I don't think that works like this

Reply:
good catch